subsys/bluetooth/common/Kconfig

```diff
@@ -60,37 +64,20 @@ config BT_BUF_ACL_RX_SIZE
 	help
 	  Maximum support ACL size of data packets sent from the Controller to
 	  the Host. This value does not include the HCI ACL header.
-	  In a combined Host and Controller build the buffer sizes in both the
-	  Host and the Controller will use this value for buffer sizes, and
-	  therefore Controller to Host flow Controller is not needed.
+
+	  In a combined Host and Controller build, both the
+	  Host and the Controller will use this value for buffer sizes, making
+	  Controller to Host flow control not strictly necessary.
+
 	  In a Host only build with Controller to Host flow control enabled
 	  the Host will inform the Controller about the maximum ACL data size it
 	  can send by setting this value in the Host Buffer Size command.
+
 	  If Controller to Host flow control is not enabled then the Controller
-	  can assume the Host has infinite buffer size so this value should then
-	  be set to something that is guaranteed the Controller will not exceed
-	  or the data packets will be dropped.
+	  can assume the Host has infinite buffers of infinite size.
```

Comment:
I would avoid this formulation 'controller can assume infinite buffers'. Not even aescolar's workstation has infinite buffers. I would prefer 'the host has receive and process ACL data faster than the controller can produce it'. And the size is also limited by the data envelope formats.